packages/kit/src/exports/vite/utils.js

```diff
@@ -90,6 +91,16 @@ export function get_config_aliases(config) {
 		{ find: '$lib', replacement: config.files.lib }
 	];
 
+	// Our generated tsconfig.json sets a path alias. This causes imports like src/utils to be valid for
+	// TypeScript (its auto completion also suggests it). However, Vite doesn't know about this,
+	// so we do a best-effort attempt to make it work by adding a Vite alias assuming the default
+	// SvelteKit project structure.
+	// For SvelteKit 2.0 we could think about getting rid of the baseUrl which would make this unnecessary
```

Comment:
maybe we could write this in a way that would be easier to find via a `grep` when v2 comes around and we want to find all the things to clean up
```suggestion
	// TODO(v2): we could think about getting rid of the baseUrl which would make this unnecessary
```